### PR TITLE
UCT/IB/LOG: Fix WQE AV logging

### PR DESCRIPTION
**Pull request by yosefe**

### CODE REVIEW

File: src/uct/ib/mlx5/ib_mlx5_log.c [hunk 1]
```diff
@@ -236,10 +236,11 @@ static size_t uct_ib_mlx5_dump_dgram(char *buf, size_t max, void *seg, int is_et
             snprintf(p, endp - p,  " sgix %d dgid %s tc %d]", sgid_index,
                      inet_ntop(AF_INET6, grh_av->rgid, gid_buf, sizeof(gid_buf)),
                      grh_av->tclass);
+        } else {
+            snprintf(p, endp - p, "]");
         }
         return UCT_IB_MLX5_AV_FULL_SIZE;
     } else {
-        snprintf(p, endp - p, "]");
```
Comment (brminich) [inline src/uct/ib/mlx5/ib_mlx5_log.c]: why need to remove here? What if we have compact av?

Comment (yosefe) [reply src/uct/ib/mlx5/ib_mlx5_log.c]: removed by mistake..

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/11636/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>UNKNOWN</b> on 17 workers (click for details)</summary>

**Note:** the logs will be deleted after 07-Aug-2019

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/5245d4d97a7a3fea9caf5249b829bd28/raw/8e2d6546169ef3cea5c2e709570de653e63c72c1/_main) | :question: ABORTED
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/5245d4d97a7a3fea9caf5249b829bd28/raw/b4d80540a36a1f88b2e67e1346f70ddef81bb8c8/hpc-arm-cavium-jenkins_W0) | :question: UNKNOWN
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/5245d4d97a7a3fea9caf5249b829bd28/raw/0a40d4d89fe1033ee8595b79353db0fff2e777b3/hpc-arm-cavium-jenkins_W1) | :question: UNKNOWN
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/5245d4d97a7a3fea9caf5249b829bd28/raw/742ca2617006b35500ec69b20b8cd0e554ed2380/hpc-arm-cavium-jenkins_W2) | :question: UNKNOWN
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/5245d4d97a7a3fea9caf5249b829bd28/raw/127b036b9676d7c2989faed679a3f54546e84c2a/hpc-arm-cavium-jenkins_W3) | :question: UNKNOWN
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/5245d4d97a7a3fea9caf5249b829bd28/raw/d9bc11e00ebd8d6957e2f291a2c051bfc2796593/hpc-arm-hwi-jenkins_W0) | :question: UNKNOWN
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/5245d4d97a7a3fea9caf5249b829bd28/raw/55893f14065e023d8bb85d8324a63f6f430b001a/hpc-arm-hwi-jenkins_W1) | :question: UNKNOWN
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/5245d4d97a7a3fea9caf5249b829bd28/raw/8ef60e6f3690c327e8dbc07956041b39938b7455/hpc-arm-hwi-jenkins_W2) | :question: UNKNOWN
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/5245d4d97a7a3fea9caf5249b829bd28/raw/397fbf90643891ab9cfd2343935070644119c901/hpc-arm-hwi-jenkins_W3) | :question: UNKNOWN
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/5245d4d97a7a3fea9caf5249b829bd28/raw/33ec5306026ed026344ea99bd2d5a9be86d4b0aa/hpc-test-node-gpu_W0) | :question: UNKNOWN
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/5245d4d97a7a3fea9caf5249b829bd28/raw/32ab0e2f0ce36e55d0439ec26d4515a53f8125e2/hpc-test-node-gpu_W1) | :question: UNKNOWN
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/5245d4d97a7a3fea9caf5249b829bd28/raw/7aececa76b8596480fe10617328c9e8ce5b2f74f/hpc-test-node-gpu_W2) | :question: UNKNOWN
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/5245d4d97a7a3fea9caf5249b829bd28/raw/5b6ba2930f99c93ea4b99d13856fd6f40dda1fa7/hpc-test-node-gpu_W3) | :question: UNKNOWN
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/5245d4d97a7a3fea9caf5249b829bd28/raw/e61599d1f70a6e37f9e7b6191cfb542cd79c4c57/hpc-test-node-legacy_W0) | :question: UNKNOWN
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/5245d4d97a7a3fea9caf5249b829bd28/raw/04fbf3ef733c181e11c2c8210169c494a42292c3/hpc-test-node-legacy_W2) | :question: UNKNOWN
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/5245d4d97a7a3fea9caf5249b829bd28/raw/2a5f5326badb8f4b0b0c315071bb4d6065fa0345/hpc-test-node-new_W2) | :question: UNKNOWN
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/5245d4d97a7a3fea9caf5249b829bd28/raw/c8c8fabb6c3a3be1017e596105e7ca23086003b5/hpc-test-node-new_W3) | :question: UNKNOWN

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/11639/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>PASSED</b#  > on 29 workers (click for details)</summary>

**Note:** the logs will be deleted after 07-Aug-2019

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/460bba94a3fee2bfb1d73cc043555b65/raw/981321499ba777f0ac946fa71de7e5a1bcd19223/_main) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/460bba94a3fee2bfb1d73cc043555b65/raw/9a631c228982002a323a916a672e023094726a5b/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/460bba94a3fee2bfb1d73cc043555b65/raw/2ebf92e77faeb8102131dda835ea1b68929a1489/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/460bba94a3fee2bfb1d73cc043555b65/raw/5c87e5e9a7732e85852f281e15f87df80d9f7a47/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/460bba94a3fee2bfb1d73cc043555b65/raw/bffa08cd9946c02c1249b57b31031a8693a27ac7/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/460bba94a3fee2bfb1d73cc043555b65/raw/7c7a7384a98bec5271f752c17a1080b38b7ab38c/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/460bba94a3fee2bfb1d73cc043555b65/raw/5daa569d3da7605cf7d9112dbe0a7466a99cc3fa/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/460bba94a3fee2bfb1d73cc043555b65/raw/0c35374fe5ea5ae5d22acb350d7f12c05a4a7b54/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/460bba94a3fee2bfb1d73cc043555b65/raw/5a260d6c83e4f152a04164e86ee0d1392c59ed8e/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W0](https://gist.githubusercontent.com/mellanox-github/460bba94a3fee2bfb1d73cc043555b65/raw/bf5f4ed9475dbcef4b5b1d4d76ec5736b722250e/hpc-test-althca_W0) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W1](https://gist.githubusercontent.com/mellanox-github/460bba94a3fee2bfb1d73cc043555b65/raw/206ebb1c67068cc8c611efebc1f847c3602e4acd/hpc-test-althca_W1) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W2](https://gist.githubusercontent.com/mellanox-github/460bba94a3fee2bfb1d73cc043555b65/raw/3229ae96caa358c4f7fa095f73616cc6a4c46c36/hpc-test-althca_W2) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W3](https://gist.githubusercontent.com/mellanox-github/460bba94a3fee2bfb1d73cc043555b65/raw/997bcf419ef7a0a6f8085912dae655cd1043785d/hpc-test-althca_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/460bba94a3fee2bfb1d73cc043555b65/raw/909001e713d037558d87f982eb292aebe3aab472/hpc-test-node-gpu_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/460bba94a3fee2bfb1d73cc043555b65/raw/f78af8aa6013c3b7761d6e4cd2f4233bd46acacb/hpc-test-node-gpu_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/460bba94a3fee2bfb1d73cc043555b65/raw/94418f8c678540fe4406a6bd5916d9ebf6cde40b/hpc-test-node-gpu_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/460bba94a3fee2bfb1d73cc043555b65/raw/4f22f9ccbf405bc85a6d9dc76286e7ff90b88696/hpc-test-node-gpu_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/460bba94a3fee2bfb1d73cc043555b65/raw/82154bbcde9a74b11998f5566d4b70dbcc097585/hpc-test-node-legacy_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/460bba94a3fee2bfb1d73cc043555b65/raw/5abc3efbb088f59158a25c26b3fef5a259bf6754/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/460bba94a3fee2bfb1d73cc043555b65/raw/aa806f0ea303b8f48394ad815c3c76537128942c/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/460bba94a3fee2bfb1d73cc043555b65/raw/fbe5cb466a682221c37ac312e61d073aa2565719/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/460bba94a3fee2bfb1d73cc043555b65/raw/5f751500c2e4c343c375ab1ca306bfd4ef2322a7/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/460bba94a3fee2bfb1d73cc043555b65/raw/775cfd267c93107e5ae344feefd3c0e0346abd79/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/460bba94a3fee2bfb1d73cc043555b65/raw/50012c88a3385bec9173d34257169893bb50598b/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/460bba94a3fee2bfb1d73cc043555b65/raw/611f07d7732875e88a251d6f7815494c29481cfc/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/460bba94a3fee2bfb1d73cc043555b65/raw/93bcb596d3b7c9d47e6ec8234318c6c1cd1a3f15/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/460bba94a3fee2bfb1d73cc043555b65/raw/d8c21b5b3cd945d504a52ec85e23267736b84610/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/460bba94a3fee2bfb1d73cc043555b65/raw/b1f421a309d3403e50f75afda935f461e7252528/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/460bba94a3fee2bfb1d73cc043555b65/raw/da9c3d78bee4ebaa78e6f2ba9dd809cb44845cc6/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

**Pull request merged**